public/app/plugins/panel/homelinks/module.tsx

```diff
@@ -71,12 +71,9 @@ export const HomeLink: FC<HomeLinkProps> = ({ title, url, target, icon }) => {
 export const VersionFooter: FC = () => {
   const styles = getStyles();
   const { version, commit } = config.buildInfo;
+  const versionString = version !== '' ? `Version ${version} (${commit})` : '';
```

Comment:
Same here - if config.hideVersion then build the version string, regardless of what's in build info.